index.js

```diff
@@ -1,34 +1,84 @@
-let editor = document.querySelector('#c');
-let error = document.querySelector("#error");
 let splitKeys = [0,1,2,3,4,5,6,7,8,9,10,11,12];
+let editor;
+let error;
 let timeout; 
-chrome.storage.sync.get(null, items => {
+let status;
+
+const init = () => {
+  document.title = 'BlinkNote';
+  editor = document.querySelector('#c');
+  error = document.querySelector("#error");
+  status = document.querySelector("#status");
+  chrome.storage.sync.get(null, items => {
+    displayText(items);
+  });
+  editor.addEventListener('keyup', (e) => { 
+    setTitle('Saving'); 
+    ([].slice.call(editor.querySelectorAll('*')).map(e => [e, e.innerText]).map(([e, text]) => [e, text.match(/^(#+)\s.*$/)]).filter(([e, m]) => m).map(([e, m]) => [e, m[1]]).forEach(([e,m]) => {e.style.transition = "font-size 0.5s cubic-bezier(0, 1.03, 1, 1) 0s"; e.style.fontSize = (30 - (3*m.length)) || 3}));
```

Comment:
@stevekrouse is this the beginning of markdown processing? I've used some libraries on NPM for this that work well.